spanner/spanner_create_instance_partition.rb

```diff
@@ -0,0 +1,54 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START spanner_create_instance_partition]
+def spanner_create_instance_partition project_id:, instance_id:, instance_partition_id:
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # instance_partition_id = "Your Spanner instance partition ID"
```

Comment:
Please update this to YARD doc format, e.g. https://github.com/aandreassa/ruby-docs-samples/blob/d74209755d7f2662ffd9e62078647e36380c64a8/spanner/spanner_alter_sequence.rb#L18-L24